bindings/wasm/examples/bindings.test.ts

```diff
@@ -0,0 +1,21 @@
+import {beforeAll, expect, suite, test} from 'vitest';
+import Module, {type ManifoldToplevel} from './built/manifold'
+
+let manifoldModule: ManifoldToplevel;
+
+beforeAll(async () => {
+  manifoldModule = await Module();
+  manifoldModule.setup();
+});
+
+suite('CrossSection Bindings', () => {
+  test('ToPolygons return correct shape', () => {
+    const polygons = manifoldModule.CrossSection.square().toPolygons();
```

Comment:
So, is the CI running this test yet? Our current JS tests are in the form of examples, but we could certainly use better coverage.

Reply:
Looking at the actions, seems like its running since vitest should should be running `*.test.*` files by default